site/content/user-guide/mqtt-adapter.md

```diff
@@ -41,6 +41,10 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
 
 After verifying the credentials, the number of existing connections is checked against the configured [resource-limits] ({{< ref "/concepts/resource-limits.md" >}}) by the MQTT adapter.  If the limit is exceeded then a return code `0x05` indicating `Connection Refused: not authorised` is sent back.
 
+## Message Limits
+
+Before accepting any telemetry or event messages, the MQTT adapter verifies that the configured [message limit] ({{< ref "/concepts/resource-limits.md" >}}) is not exceeded. The incoming message is declined if the limit is exceeded. 
```

Comment:
please document the adapter's behavior in the sections on publishing telemetry/event messages.

Reply:
The last sentence should probably be changed to
> The incoming message is discarded if the limit is exceeded.

Reply:
done.